charts/karmada/_crds/bases/work/work.karmada.io_works.yaml

```diff
@@ -20,6 +20,9 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
+    - jsonPath: .spec.workload.manifests[*].kind
```

Comment:
What's the difference between `manifests[*]` and `manifests[0]`?

Reply:
It looks the same, that only print the fisrt one.